packages/studio-be/src/core/misc/archive.ts

```diff
@@ -1,6 +1,60 @@
+import fse from 'fs-extra'
```

Comment:
Can you add at least unit tests for the two files in the misc folder.

Reply:
Not touching archive, it was copied from the main repo. Added a comment